<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTreatmentForm</class>
 <widget class="QWidget" name="MainTreatmentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>362</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="mta_volume_label">
       <property name="text">
        <string notr="true">Volume</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mta_viscosity_label">
       <property name="text">
        <string notr="true">Viscosity</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mta_pumping_pressure_label">
       <property name="text">
        <string notr="true">Pumping pressure</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mta_pumping_rate_label">
       <property name="text">
        <string notr="true">Pumping rate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="mta_volume_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="mta_volume_unit_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="mta_viscosity_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="mta_viscosity_unit_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>P</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="mta_pumping_pressure_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="mta_pumping_pressure_unit_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pa</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="mta_pumping_rate_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="mta_pumping_rate_unit_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/s</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="mta_density_label">
       <property name="text">
        <string notr="true">Density</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mta_treatment_label">
       <property name="text">
        <string notr="true">Treatment</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mta_treatment_lineedit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mta_pumping_treatment_label">
       <property name="text">
        <string notr="true">Pumping treatment</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="mta_density_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="mta_density_unit_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>kg/m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="mta_pumping_treatment_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="mta_pumping_treatment_unit_label">
         <property name="text">
          <string>m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/s</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="mta_additives_label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string notr="true">Additives</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="mta_additives_tableview">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
